apps/dashboard/src/routes/+layout.svelte

```diff
@@ -29,16 +33,32 @@
 <AppShell>
 	<svelte:fragment slot="header">
 		<AppBar>
-			<svelte:fragment slot="lead"><a href="/">Turtle Links</a></svelte:fragment>
-			<svelte:fragment slot="trail"
-				><button type="button" class="btn variant-filled-primary">Post Item</button>
+			<svelte:fragment slot="lead"
+				><a href="/" class="text-xl font-bold"
+					><button type="button" class="btn variant-filled-surface">
+						<span><Fa icon={faLink} /></span>
+						<span><span class="text-green-400">Turtle</span> Links</span>
+					</button></a
+				></svelte:fragment
+			>
+
+			<svelte:fragment slot="trail">
+				<a href="https://github.com/chandertech/turtlelinks"
+					><button type="button" class="btn-icon variant-filled-surface">
+						<Fa icon={faGithub} />
+					</button></a
+				>
 				<a href="/login"
-					><Avatar
-						src="https://images.unsplash.com/photo-1617296538902-887900d9b592?ixid=M3w0Njc5ODF8MHwxfGFsbHx8fHx8fHx8fDE2ODc5NzExMDB8&ixlib=rb-4.0.3&w=128&h=128&auto=format&fit=crop"
-						width="w-8"
-						rounded="rounded-full"
-					/>
-				</a>
+					><button type="button" class="btn variant-filled-surface">
+						{#if data.session}
```

Comment:
Should I setup a store for this? I imagine there might be a few places where we want to check if a user is logged in. (Assuming that store is the correct use for what I want here)

Reply:
Have you seen a Supabase project use a Store for this?

Reply:
nope, was just an idea that I had.

Reply:
Kk lets keep and eye on it and see how it would work as we add more pages